api/SourceBuffer.json

```diff
@@ -583,7 +583,7 @@
               }
             ],
             "edge": {
-              "version_added": true
+              "version_added": "18"
```

Comment:
Getting an instance of `SourceBuffer` takes some work, but confirmed by `SourceBuffer.prototype.hasOwnProperty('textTracks')` being true in Edge 18 and false in Edge 17.